File: go/go-go-fuzz/fuzzme.go
package test

import (
    "fmt"
    "io/ioutil"
    "log"
    "os"
)

func test() {
    if len(os.Args) != 2 {
        fmt.Printf("Usage: ./fuzzme <file>\n")
        os.Exit(1)
    }
    bytes, err := ioutil.ReadFile(os.Args[1])
    if err != nil {
        log.Fatal(err)
    }
    Bad(bytes)
}

func Bad(bytes []byte) int {
    content := string(bytes)
    if len(content) < 3 {
        return 0
    }
    if string(content[0]) == "b" {
        if string(content[1]) == "u" {
            if string(content[2]) == "g" {
                _ = divBy(0)
            }
        }
    }
    return 0
}

func divBy(val int) int {
    y := 1 / val
    return y
}

func Fuzz(data []byte) int {
    _ = Bad(data)
    return 0
}

File: java/libfuzzer/java-jazzer/src/FuzzMe.java
package fuzzme;

public class FuzzMe {
    public static void fuzzerTestOneInput(byte[] data) {
        String input = new String(data);
        if (input.length() >= 3) {
            if (input.charAt(0) == 'b') {
                if (input.charAt(1) == 'u') {
                    if (input.charAt(2) == 'g') {
                        System.exit(0);
                    }
                }
            }
        }
    }
}
